pemFioi/quickpi/blocklyQuickPi_lib.js

```diff
@@ -6678,7 +6678,7 @@ var getContext = function (display, infos, curLevel) {
                     cliph
             });
 
-            sensor.stateText = paper.text(state1x, state1y, sensor.state + "C");
+            sensor.stateText = paper.text(state1x, state1y, sensor.state + "°C");
```

Comment:
There should be a space before °C as well (there's no space only if it's for trigonometry degrees).